File: Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace dpfinal_200042152
{
    public class Comment
    {
        public int Id;
        public string Content;
        public string Author;
        public DateTime CreatedAt;


        public Comment(int _id, string _content,string _author, DateTime _createdAt) 
        { 
        
            Id = _id;
            Content = _content;
            Author = _author;
            CreatedAt = _createdAt;
        
        }
         
        public Comment(Comment original) 
        
        { 
            Id=original.Id;
            Content = original.Content;
            Author = original.Author;
            CreatedAt = original.CreatedAt;
        
        }
    }
}

File: TaskManager/210042141_TaskManager/src/Comment.java
import java.util.Date;

public class Comment {
    public int id;
    public String content;
    public String author;
    public Date createdAt;

    public Comment(int id, String content, String author, Date createdAt) {
        this.id = id;
        this.content = content;
        this.author = author;
        this.createdAt = createdAt;
    }

}
